src/client/styles/components/_form-dropdown.scss

```diff
@@ -7,7 +7,7 @@
   padding: $baseline-grid-unit;
   width: 100%;
   line-height: 32px;
-  font-size: 30px;
+  font-size: 20px;
```

Comment:
I have few commented code kept for my reference.
Once all the changes are done for nested steps, I will remove the comments.